include/circt/Conversion/Passes.td

```diff
@@ -374,6 +374,8 @@ def LowerFIRRTLToHW : Pass<"lower-firrtl-to-hw", "mlir::ModuleOp"> {
             "Disable emission of memory randomization code">,
     Option<"disableRegRandomization", "disable-reg-randomization", "bool", "false",
             "Disable emission of register randomization code">,
+    Option<"initializeMemSynthesis", "initialize-mem-synthesis", "bool", "false",
+            "Enable memory initialization when SYNTHESIS flag is defined by synthesizer (Eg. FPGAs)">,
```

Comment:
Based on the emission, this is really "unconditionally initialize the memory" not "enable when SYNTHESIS is set", yes?

Ditto for other descriptions.

Reply:
What about "Enable unconditional memory initialization (Eg. when SYNTHESIS flag is defined by synthesis tool)"?
Too descriptive?